doc/NOW-spec.md

```diff
@@ -973,17 +1014,13 @@ The NOW_EXEC_DATA_MSG message is used to send input/output data as part of a rem
 
 | Flag                                   | Meaning                         |
 |----------------------------------------|---------------------------------|
-| NOW_EXEC_FLAG_DATA_FIRST<br>0x00000001 | This is the first data message. |
```

Comment:
`first` flags for streams is not necessary and only complicates data API/protocol